javaslang-benchmark/src/test/java/javaslang/benchmark/MicroBenchmark.java

```diff
@@ -1,30 +1,19 @@
 package javaslang.benchmark;
 
-import javaslang.collection.List;
-import javaslang.collection.Queue;
-import javaslang.collection.Stream;
-import javaslang.collection.Vector;
+import javaslang.collection.*;
 import org.junit.Test;
 import org.openjdk.jmh.annotations.*;
-import org.openjdk.jmh.infra.Blackhole;
 import org.openjdk.jmh.runner.Runner;
-import org.openjdk.jmh.runner.options.Options;
-import org.openjdk.jmh.runner.options.OptionsBuilder;
-import org.openjdk.jmh.runner.options.TimeValue;
+import org.openjdk.jmh.runner.options.*;
 
-
-import java.util.Random;
 import java.util.concurrent.TimeUnit;
 
 @State(Scope.Benchmark)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @BenchmarkMode(Mode.Throughput)
 public class MicroBenchmark {
```

Comment:
Sure, good idea, please commit your benchmarks and I'll extend it :)

Also note that there is an exiting issue about benchmarks: https://github.com/javaslang/javaslang/issues/725

Would be awesome to find out where the actual optimizations would be needed :).
Maybe we could do the benching in the way the tests are organized, i.e. polymorphically instead of hard coding every method separately.

@eduardmanas, @danieldietrich, @ruslansennov, opinion?